R/parse.R

```diff
@@ -111,7 +111,7 @@ get_parse_data <- function(text, include_text = TRUE, ...) {
   pd <- pd %>%
     add_id_and_short()
 
-  parser_version_set(parser_version_find(pd))
+  # parser_version_set(parser_version_find(pd))
```

Comment:
@lorenzwalthert Do we still want to create environment bindings tracking parser version and retain the functions to do so?

Once we bump to `R >= 3.6`, they are no longer used or needed anywhere, so I am wondering if it's worth retaining them.

Reply:
WDYT, @lorenzwalthert?

Reply:
Yes, you are right. I thought we might still need to distinguish because there is also version 3 of the parser but that's not needed anymore.

Reply:
Question is if we should remove the whole parser versioning logic or not, since in the future, we might still want to distinguish between parser versions, as new changes in the parser might require dispatch.

Reply:
Yeah, let's retain this logic even if we don't need it currently, just to be future-proof.